astro/src/content/docs/apis/_import-users-request-body.mdx

```diff
@@ -338,12 +338,8 @@ You must provide either the **email** or the **username** field for each User. T
   <ModerationStatusRequest />
 </APIField>
 
-<APIField name="users[x].verified" type="Boolean" optional defaults="false">
```

Comment:
Note that we had not removed this yet (forgot). But this is not mutable anymore.

Reply:
Per https://github.com/FusionAuth/fusionauth-issues/issues/3070, it wasn't ever really possible though, although for importing, you could, and now you can't.

Reply:
Is this breaking?

Reply:
If it is, we probably should add a piece of work to add some tests+implementation for this (and `user.verifiedInstant`) to ensure that we set values on the primary email identity instead when the user level fields are used.

Reply:
Note that in 1.58.2, user create and user update API operations always ignore `user.verified` on the way in. It's 100% derived, except in import cases, as noted above.

Reply:
@andrewpai note this has been fixed